public/app/plugins/panel/stat/override.ts

```diff
@@ -0,0 +1,93 @@
+import { FieldDisplay, FieldConfigEditorBuilder, InterpolateFunction, GrafanaTheme2, FieldType } from '@grafana/data';
+
+import { PanelFieldConfig } from './panelcfg.gen';
+
+export function addDisplayOverrideOptions(builder: FieldConfigEditorBuilder<PanelFieldConfig>) {
+  const category = ['Override display'];
+  // This value indicates that the option should not be available in the Field config tab
+  const hideFromDefaults = true;
+  return builder
+    .addTextInput({
+      path: 'prefix',
+      name: 'Prefix',
+      defaultValue: undefined,
+      hideFromDefaults,
+      settings: {
+        placeholder: 'Override prefix',
+      },
+      category,
+    })
+    .addTextInput({
+      path: 'suffix',
+      name: 'Suffix',
+      defaultValue: undefined,
+      settings: {
+        placeholder: 'Override suffix',
+      },
+      category,
+      hideFromDefaults,
+    })
+    .addTextInput({
+      path: 'text',
+      name: 'Text',
+      defaultValue: undefined,
+      settings: {
+        placeholder: 'Override text',
+      },
+      category,
+      hideFromDefaults,
+    })
+    .addColorPicker({
+      path: 'color',
+      name: 'Color',
+      defaultValue: undefined,
+      settings: {
+        placeholder: 'Custom color',
+        isClearable: true,
+      },
+      category,
+      hideFromDefaults,
+    });
+}
+
+export function applyDisplayOverrides(
+  disp: FieldDisplay,
+  theme: GrafanaTheme2,
+  replace: InterpolateFunction,
+  applyToType: FieldType[]
+): FieldDisplay {
+  const { valueType } = disp.display;
+  // Only apply the display overrides if the valueType (FieldType) has been specified
+  if (applyToType.some((type) => type === valueType)) {
```

Comment:
why do we need this?   you can specify what fields you want the overrides to apply to in normal override path?

Reply:
Ah, I see now; sorry! Removed.